src/plugins/intel_gna/tests/functional/pass_tests/remove_permutations_NHWC_to_NCHW_pass.cpp

```diff
@@ -649,7 +649,9 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::
                                                                InferenceEngine::Precision::FP16};
 
 const std::vector<std::map<std::string, std::string>> configs = {
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}},
+    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}, {"GNA_EXEC_TARGET", "GNA_TARGET_2_0"}},
+    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_0"}},
+    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_SCALE_FACTOR_0", "327.67"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_5"}},
     {{"GNA_DEVICE_MODE", "GNA_SW_FP32"}}};
 
```

Comment:
you would need to add new explicit target - this is related to the our functionality if you use GNA_SW_EXACT without explicitly giving GNA_EXEC_TARGET it will use default one in case HW is not present. If hw is present it will use hardware version.

So if you want to check if there is no regressions for older platforms and ensure that tests are run on new platforms this is required.

Reply:
then - acting this way - for all tests with GNA_DEVICE_MODE=GNA_SW_EXACT where GNA_EXEC_TARGET is NOT enumerated, we are not tracking regressions per target, UNLESS problem emerges for  default or real hardware target.

Reply:
I have removed targets